vmc/resource_vmc_sddc.go

```diff
@@ -171,9 +171,29 @@ func resourceSddc() *schema.Resource {
 				Type:     schema.TypeMap,
 				Computed: true,
 			},
+			"availability_zones": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
 		},
 		CustomizeDiff: func(d *schema.ResourceDiff, meta interface{}) error {
 
+			deploymentType := d.Get("deployment_type").(string)
+			numHosts := d.Get("num_host").(int)
+
+			if deploymentType == "MultiAZ" && numHosts < 6 {
```

Comment:
use constant instead of string literal for MultiAZ

Reply:
the min number of hosts needed should also be a constant. What would that number be if we allow 2 host SDDC?

Reply:
For SingleAZ the min number of hosts is 1 (validation for that is already handled using validation function on num_host field). For MultiAZ the minimum number of hosts is 6 which I can define as a constant in the constants.go

Reply:
Done

Reply:
I dont see the usage of the constant for minimum no of hosts.

Reply:
https://github.com/terraform-providers/terraform-provider-vmc/pull/35/files#diff-d6903083e67381c2ae26986e118c9d59R185 This is the latest commit